spot_driver/spot_driver/ros_helpers.py

```diff
@@ -262,6 +264,121 @@ def bosdyn_data_to_image_and_camera_info_msgs(data: image_pb2.ImageResponse, rob
 
     return image_msg, camera_info_msg
 
+def bosdyn_localization_to_pose_msg(
```

Comment:
conversion functions should live in `manual_conversions.py`

Reply:
I wasn't aware of `manual_conversions`. I see now - it's another file under spot_driver.

Reply:
I was adding these methods in ros_helpers because I saw these other methods like `bosdyn_data_to_image_and_camera_info_msgs`
https://github.com/bdaiinstitute/spot_ros2/blob/main/spot_driver/spot_driver/ros_helpers.py#L199

I looked at manual_conversions, but I think the ones in this PR are new (e.g. there's no conversion to TransformStamped). But I could move these functions there.

Reply:
I attempted the move and then I realize, `spot_ros2` isn't using anything from manual_conversions.

Reply:
I moved them to manual_conversions.